styleguide/components/ComponentsList/index.js

```diff
@@ -43,16 +43,19 @@ export function ComponentsList({ classes, items }) {
                 css={css`
                   color: ${theme.colors.neutral.grey800} !important;
                   width: 100% !important;
-                  padding: 16px 0 !important;
+                  padding: ${theme.spacing.padding.m} 0 !important;
                   border-top: 2px solid !important;
                   display: block !important;
+                  text-transform: uppercase !important;
```

Comment:
Brute-forced, eh?

Reply:
Yeah, we might need to rebuild the ComponentList component so we don't need to do this nonsense, but for now I'm !important flagging like it's going out of style.